### DISCUSSION TITLE
Có dịch "K-Means" không?

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#79] [Terminology] [answered]
@rootonchair đang dịch là `K-Trung bình`. Mn thấy sao?
cc: @mlbvn/handson-ml2

**Comment by lkhphuc** [comment]
Nếu dịch thì nên là `K-điểm trung bình`.

**Reply by cuongvng** [reply]
em thấy hơi dài, vote giữ nguyên.

**Reply by thanhcsf** [reply]
Một cách OCD nhất thì anh cũng muốn dịch ra như thế này.

**Comment by minhduc0711** [comment]
Mình cũng thích để nguyên từ này. Anh @ngcthuong thấy sao ạ?

**Comment by minhduc0711** [comment]
pinggg

**Reply by ngcthuong** [reply]
Từ này cũng là từ căn bản, nếu dịch thì mình cũng đồng ý là "K-điểm trung bình".

**Comment by minhduc0711** [comment, marked as answer]
Chốt là `k-điểm trung bình` nhé mọi người.
